benchmarks/benchmark_ocl.py

```diff
@@ -0,0 +1,161 @@
+import tempfile
+from pathlib import Path
+
+from kedro.config import OmegaConfigLoader
+
+base_catalog = {
+    "dataset_1": {
+        "type": "pandas.CSVDataset",
+        "filepath": "data1.csv"
+    },
+    "dataset_2": {
+        "type": "pandas.CSVDataset",
+        "filepath": "data2.csv"
+    },
+    "dataset_3": {
+        "type": "pandas.CSVDataset",
+        "filepath": "data3.csv"
+    },
+    "dataset_4": {
+        "type": "pandas.CSVDataset",
+        "filepath": "data4.csv",
+        "versioned": True,
+    },
+}
+local_catalog = {
+    "dataset_4" : {
+        "filepath": "data4_local.csv",
+        "type": "pandas.CSVDataset",
+    },
+    "dataset_5" : {
+        "filepath": "data5_local.csv",
+        "type": "pandas.CSVDataset",
+    },
+}
+base_params = {
+    "param_1": "value_1",
+    "param_2": "value_2",
+    "param_3": "value_3",
+    "param_4": "value_4",
+}
+local_params = {
+    "param_4": "value_4_local",
+    "param_5": "value_5_local",
+}
+base_globals = {
+    "global1": "value1",
+    "global2": "value2",
+    "global3": "value3",
+    "global4": "value4",
+}
+local_globals = {
+    "global4": "value4_local",
+    "global5": "value5_local",
+}
+
+def _create_config_file(self, env, file_name, data):
+        env_path = self.conf_source / env
+        env_path.mkdir(parents=True, exist_ok=True)
+        file_path = env_path / file_name
+
+        import yaml
+        with open(file_path, "w") as f:
+            yaml.dump(data, f)
+
+
+class TimeOmegaConfigLoader:
+
+    def setup(self):
+        # Setup temporary configuration directory with sample config files
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.conf_source = Path(self.temp_dir.name)
```

Comment:
It seems that `asv` already create some temp dir when I run test, do we need to create a separate one ourself?

Reply:
This is to create a temporary conf directory structure for the config for each test